packages/wdio-config/src/constants.js

```diff
@@ -74,3 +74,10 @@ export const SUPPORTED_HOOKS = [
     'beforeFeature', 'beforeScenario', 'beforeStep', 'afterStep', 'afterScenario', 'afterFeature',
     'onReload', 'onPrepare', 'onComplete'
 ]
+
+/**
+ * these services should not be started in worker process
+ */
+export const NON_WORKER_SERVICES = [
```

Comment:
Instead of having to maintain a list like that can we just check after we required the launcher whether the service also offers a worker service?

Reply:
Not sure how to achieve that. I mean we can check everything after requiring the launcher, however the goal is to avoid importing services so that they won't load any modules, as a result I can't import launcher. 

Maybe in v6 we can change the way how services are loaded?

Reply:
> Maybe in v6 we can change the way how services are loaded?

Yeah. I guess what I suggested is just more of a hacky workaround. Let's use v6 to introduce a clear way to determine whether or not a service has to be also get required in a worker process.